client-app/package.json

```diff
@@ -30,17 +30,19 @@
     "jest-environment-jsdom": "^29.7.0",
     "jest-localstorage-mock": "^2.4.26",
     "jsonwebtoken": "^9.0.2",
-    "mdbreact": "^5.2.0",
+    "mdbreact": "^4.6.1",
     "mongoose": "^8.1.2",
     "my-tailwind-app": "^0.1.12",
     "nodemailer": "^6.9.10",
-    "npx": "^10.2.2",
+    "npx": "^3.0.0",
     "path-browserify": "^1.0.1",
     "postcss": "^8.4.35",
+    "prettier": "^3.4.1",
     "react": "^18.2.0",
     "react-barcode": "^1.5.1",
     "react-dom": "^18.2.0",
     "react-icons": "^5.0.1",
+    "react-modal": "^3.16.1",
     "react-router-dom": "^6.22.1",
```

Comment:
I think this issue do not need any of these dependency updates. Please revert it

Reply:
Also revert package-lock.json file also